Sources/SwiftInspectorTestHelpers/SyntaxVisitor+WalkContent.swift

```diff
@@ -25,21 +25,13 @@
 import Foundation
 import SwiftSyntax
 
-public final class VisitorExecutor {
-
-  /// Walks the provided visitor along the content's syntax.
+extension SyntaxVisitor {
+  /// Walks the visitor along the content's syntax.
   ///
   /// - Parameters:
-  ///   - visitor: The visitor to walk along the content syntax.
   ///   - content: The content to turn into source and walk.
-  public static func walkVisitor<Visitor: SyntaxVisitor>(
-    _ visitor: Visitor,
-    overContent content: String)
-  throws
-  {
-    let fileURL = try Temporary.makeFile(content: content)
-    let syntax: SourceFileSyntax = try SyntaxParser.parse(fileURL)
-    visitor.walk(syntax)
-    try Temporary.removeItem(at: fileURL)
+  public func walkContent(_ content: String) throws {
+    let syntax: SourceFileSyntax = try SyntaxParser.parse(source: content)
```

Comment:
I don't love that we can't write a test for this new method but... I guess everything will fail if this fails

Reply:
I'm fine not writing a test for this since that'd test that `SyntaxParser` is working appropriately, which we shouldn't really test IMO.